packages/extensions-sdk/src/cli/commands/add.ts

```diff
@@ -175,8 +174,6 @@ export default async function add(options: AddOptions): Promise<void> {
 			process.exit(1);
 		}
 
-		const oldName = extensionManifest.name.match(EXTENSION_NAME_REGEX)?.[1] ?? extensionManifest.name;
-
```

Comment:
I think it would make sense to revert this change, following 35714f6a4efe67d071df3de9d6bc8a814bded5dd

Reply:
I'm not sure if that's relevant anymore, as it's only the package extensions that are checking the name now 🤔 This is only for adding entries to a bundle IIRC